inc/Engine/Admin/Settings/Page.php

```diff
@@ -2145,4 +2154,16 @@ private function disable_combine_js(): bool {
 
 		return ! (bool) get_rocket_option( 'minify_js', 0 );
 	}
+
+	/**
+	 * Set optimize css delivery method value.
+	 *
+	 * @since 3.9.3
```

Comment:
```suggestion
	 * @since 3.10
```

This UI change will be in the next major version